Alcolator/Alcolator/Beer.h

```diff
@@ -0,0 +1,16 @@
+//
+//  Beer.h
+//  Alcolator
+//
+//  Created by Ryan Walker on 9/30/15.
+//  Copyright © 2015 Ryan Walker. All rights reserved.
+//
+
+#import "AlcoholicBeverage.h"
+
+@interface Beer : AlcoholicBeverage
+
+- (NSString *)vesselDescriptor:(int)numberOfBeers;
```

Comment:
If you make this method signature more like this:
```
- (NSString *)vesselDescriptorForQuantity:(NSNumber *)quantity;
```
you wouldn't have to re-declare it in the `Beer.h` and `Wine.h`. The quantity and `NSNumber` part of it make it generic for those subclasses.